tests/libtransmission/crypto-test.cc

```diff
@@ -48,25 +48,25 @@ std::string toString(std::array<std::byte, N> const& array)
 
 TEST(Crypto, DH)
 {
-    auto a = tr_message_stream_encryption::DH{};
-    auto b = tr_message_stream_encryption::DH{};
+    auto a = tr_message_stream_encryption::DH{ tr_message_stream_encryption::DH::randomPrivateKey() };
```

Comment:
Is `tr_message_stream_encryption::DH::randomPrivateKey()` used anywhere outside this test now?

Reply:
Yes.

https://github.com/transmission/transmission/blob/235736dcefa697bba4c1cc1ece119962bbb5950e/libtransmission/handshake.h#L77-L80